src/libraries/System.Private.Xml/src/System/Xml/Serialization/SoapSchemamember.cs

```diff
@@ -1,14 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
 namespace System.Xml.Serialization
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;
     using System.Xml;
 
     public class SoapSchemaMember
     {
-        private string _memberName;
+        private string? _memberName;
         private XmlQualifiedName _type = XmlQualifiedName.Empty;
 
         public XmlQualifiedName MemberType
```

Comment:
Actually while me and Carlos were discussing this, he pointed out that if someone writes `SoapSchemaMember.MemberType = null`, the compiler will complain becuase `_type` is now a non-nullable reference type.

Reply:
Good point, will change, I marked this as non-null because it was assigned to non-null property but it seems that one actually was missing `[AllowNull]`